util.go

```diff
@@ -41,3 +42,8 @@ func xtob(x1, x2 byte) (byte, bool) {
 	b2 := xvalues[x2]
 	return (b1 << 4) | b2, b1 != 255 && b2 != 255
 }
+
+// Compare compare two uuids
```

Comment:
`// Compare returns an integer comparing two uuids lexicographically. The result will be 0 if a == b, -1 if a < b, and +1 if a > b.`

Reply:
ok , done,
thank you!